nvflare/private/fed/server/job_cmds.py

```diff
@@ -165,12 +166,17 @@ def get_spec(self):
         )
 
     def authorize_job_file(self, conn: Connection, args: List[str]):
-        if len(args) < 2:
+        """
+        Args: cmd_name tx_id job_id file_name [end]
+        """
+        if len(args) < 3:
             conn.append_error(
                 "syntax error: missing job_id", meta=make_meta(MetaStatusValue.SYNTAX_ERROR, "missing job_id")
             )
```

Comment:
This error message does not match the check "if len(args) <3"
Users could be missing one of the following: tx_id, job_id or file_name.

Reply:
Good catch! Will fix.

Reply:
Fixed.